recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
This runs the publishing pipeline again and builds a new py36 version. I thought without the bump I would run into problems uploading to Anaconda.